usaspending_api/settings.py

```diff
@@ -80,6 +80,7 @@
 BULK_DOWNLOAD_S3_REDIRECT_DIR = "generated_downloads"
 BULK_DOWNLOAD_SQS_QUEUE_NAME = ""
 DATABASE_DOWNLOAD_S3_REDIRECT_DIR = "database_download"
+DATABASE_DOWNLOAD_S3_BUCKET_NAME = CONFIG.DATABASE_DOWNLOAD_S3_BUCKET_NAME
```

Comment:
Any reason this needs to be added to settings and can't be accessed via config?

Reply:
I suggested adding the bucket name in the settings file to keep with our precedent of defining these config variable in the `settings.py` file. It looks like we import from `CONFIG` more than I originally thought, so maybe that's not as much of a precedent as I was thinking.